src/patternfly/components/Popover/examples/Popover.md

```diff
@@ -132,34 +140,126 @@ import './Popover.css'
 {{/popover}}
 ```
 
+### Variations
+
+```hbs
+{{#> popover popover--modifier="pf-m-top pf-m-success" popover--attribute='aria-labelledby="popover-top-header" aria-describedby="popover-top-body"'}}
+  {{#> popover-content}}
+    {{#> button button--modifier="pf-m-plain" button--attribute='aria-label="Close"'}}
+      <i class="fas fa-times" aria-hidden="true"></i>
+    {{/button}}
+    {{#> popover-icon popover-icon--type="check-circle"}}
+    {{/popover-icon}}
+    {{#> title titleType="h1" title--modifier="pf-m-md pf-m-success" title--attribute='id="popover-top-header"'}}
+      Success Popover header
+    {{/title}}
+    {{#> popover-body popover-body--attribute='id="popover-top-body"'}}
```

Comment:
Since this generates an `id` attribute on this element, it needs to be unique on the page, and there is already an example with the name "popover-top-body" on this page, which is generating the a11y error in the PR build.

```suggestion
    {{#> popover-body popover-body--attribute='id="popover-success-alert-body"'}}
```

Reply:
This is fixed now.